Documentation/boards/raspberrypi/README.md

```diff
@@ -1,15 +1,17 @@
 # Raspberry PI
 
-Supported Hardware:
-
-| Device | Board |
-|--------|-----------|
-| Raspberry Pi A+/B/B+| rpi |
-| Raspberry Pi Zero | rpi |
-| Raspberry Pi Zero W | rpi0-w |
-| Raspberry Pi 2 B | rpi2 |
-| Raspberry Pi 3 B/B+ | rpi3 / rpi3-64 |
-| Raspberry Pi 4 B | rpi4 / rpi4-64 |
+## Supported Hardware
+
+| Device              | Release Date  | Support         | Config             |
+|---------------------|---------------|-----------------|--------------------|
+| Raspberry Pi B/B+/A+|2012/2014/2014 | not recommended | [rpi](../../../buildroot-external/configs/rpi_defconfig)              |
+| Raspberry Pi 2 B    |2015           | not recommended | [rpi2](../../../buildroot-external/configs/rpi2_defconfig)             |
+| Raspberry Pi Zero   |2015           | not recommended | [rpi](../../../buildroot-external/configs/rpi_defconfig)              |
+| Raspberry Pi Zero W |2017           | not recommended | [rpi0_w](../../../buildroot-external/configs/rpi0_w_defconfig)           |
+| Raspberry Pi 3 B/B+ |2016/2018      | yes             | [rpi3](../../../buildroot-external/configs/rpi3_defconfig) / [rpi3_64](../../../buildroot-external/configs/rpi3_64_defconfig) |
+| Raspberry Pi 4 B    |2019           | yes*            | [rpi4](../../../buildroot-external/configs/rpi4_defconfig) / [rpi4_64](../../../buildroot-external/configs/rpi4_64_defconfig) |
+
+\*1,2 and 4 GiB versions of the Raspberry Pi 4 B are supported. The 8 GiB version is not yet supported
```

Comment:
This changed very recently. I guess you need to rebase/merge the latest changes from the dev branch. But maybe its better if this gets merged first.

Reply:
Made reference to this in #77ffa1f